gui/src/components/dataOutputConfigs/DataOutputConfigsMain.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+
+class DataOutputConfigsMain extends Component {
+
+  render() {
+    return (
+      <div className="container">
+        <h1>Your Data Output Configs</h1>
```

Comment:
I would use full version of words when they are displayed to the user. I would change "Configs" to Configurations.